onnx/numpy_helper.py

```diff
@@ -205,6 +205,27 @@ def float8e5m2_to_float32(
     return res.reshape(dims)  # type: ignore[no-any-return]
 
 
+def float8e8m0_to_float32(
```

Comment:
```suggestion
```

I would remove this function. Consider moving to test if it is only used there. You can see the ones above are marked deprecated

Reply:
Yes that's the intention. Could you comment on https://github.com/onnx/onnx/issues/6605 if it sounds good to you?